src/Microsoft.OpenApi.OData.Reader/Edm/ODataSegment.cs

```diff
@@ -100,6 +102,17 @@ public string GetPathItemName(OpenApiConvertSettings settings)
         {
             return GetPathItemName(settings, new HashSet<string>());
         }
+        /// <summary>
+        /// Profides a suffix for the operation id based on the operation path.
+        /// </summary>
+        /// <param name="path">Path to use to deduplicate.</param>
+        /// <param name="settings">The settings.</param>
+        ///<returns>The suffix.</returns>
+        public string GetPathHash(OpenApiConvertSettings settings, ODataPath path = default)
```

Comment:
no added into the public api?

Reply:
it was, unless I missed something? https://github.com/microsoft/OpenAPI.NET.OData/pull/149/files#diff-2b4bfbc25273377536cf444393058ae0e6da90188a25e7cfb19caff894997678R70